crates/symbolicator/src/config.rs

```diff
@@ -366,6 +368,44 @@ impl Config {
     }
 }
 
+#[derive(Debug)]
+struct LevelFilterVisitor;
```

Comment:
an alternative would be to define our own enum for this with a derived `Deserialize`, and convert between these two directly.